django_jinja/views/__init__.py

```diff
@@ -46,22 +47,24 @@ def put(self, request, *args, **kwargs):
     def patch(self, request, *args, **kwargs):
         return self.get(request, *args, **kwargs)
 
+    match_extension = get_match_extension() or ".jinja"
```

Comment:
I think that this should be resolved at runtime and not at class definition time. With your current approach can execute code before django completly initializes...